Cron/Feed.php

```diff
@@ -132,6 +132,9 @@ public function execute()
                 $products = $this->getVisibleProducts($storeId, $page);
                 $count = count($products);
 
+                $object_manager = \Magento\Framework\App\ObjectManager::getInstance();
+                $stock_state = $object_manager->get('\Magento\CatalogInventory\Api\StockStateInterface');
```

Comment:
Does this need to be defined on each loop?

Reply:
Nope.